data/dev-rakeshshaw.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rakesh Shaw",
+  "type": "personal",
+  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience."
+  "avatar": "https://github.com/dev-rakeshshaw.png"
```

Comment:
Hi @dev-rakeshshaw  You can remove the avatar, There's no need to add it.
rest looks good 👍🏼

Reply:
Can you please guide me on how can I remove the avatar?
Actually, I am new to this type of thing.

Reply:
> Can you please guide me on how can I remove the avatar? Actually, I am new to this type of thing.

hey @dev-rakeshshaw just go `File Changed` tab 
![Screenshot from 2023-02-14 19-38-50](https://user-images.githubusercontent.com/88102392/218762275-d613e2b7-fec3-4189-a4c0-56f40935fdc0.png)
Click on dot menu > edit file & remove the line
![Screenshot from 2023-02-14 19-39-34](https://user-images.githubusercontent.com/88102392/218762479-49cf2351-1f9f-41bc-aeee-d4003bb451ec.png)

Reply:
Thanks, brother.